### DISCUSSION TITLE
install  issue

### DISCUSSION THREAD

**Discussion by top58525785** [frangoteam/FUXA#565] [General]
HI FUXA Team:
When i download V1.1.9 version
I meet this install issues
Environment:WIN10
Using : Install NPM
These error happens
So Cloud you tell how to fix?
![error install](https://user-images.githubusercontent.com/54358179/198226744-c6f10ce0-c709-48cd-b7ad-b4489f2f6e24.png)
Thanks a lot

**Comment by malcore10** [comment]
Hi, nodejs version? [You need to have installed Node (Version 10, 12 or 14) and NPM (Version >=6.11 and < 7)](https://github.com/frangoteam/FUXA/wiki/Installing-and-Running)

**Comment by top58525785** [comment]
Thanks for reply
I used V1.16 is ok
but 1.1.9 failed,NPM and node is alreadly installed
Here is the log file
Could you help me ?
[2022-10-27T15_54_44_602Z-debug-0.log](https://github.com/frangoteam/FUXA/files/9881048/2022-10-27T15_54_44_602Z-debug-0.log)

**Comment by top58525785** [comment]
Is there anyone to help me?:(

**Reply by unocelli** [reply]
Hi, you should try nodejs version 14 [wiki](https://github.com/frangoteam/FUXA/wiki/Installing-and-Running)
